frontend/javascripts/oxalis/view/action-bar/tracing_actions_view.tsx

```diff
@@ -461,16 +475,11 @@ class TracingActionsView extends React.PureComponent<Props, State> {
       activeUser,
       layoutMenu,
       busyBlockingInfo,
-      othersMayEdit,
+      isAnnotationLockedByUser,
       annotationOwner,
     } = this.props;
-    const copyAnnotationText =
-      !restrictions.allowUpdate &&
-      activeUser != null &&
-      annotationOwner?.id === activeUser.id &&
-      othersMayEdit
-        ? "Duplicate"
-        : "Copy To My Account";
+    const isAnnotationOwner = activeUser && annotationOwner?.id === activeUser?.id;
+    const copyAnnotationText = isAnnotationOwner ? "Duplicate" : "Copy To My Account";
     const archiveButtonText = task ? "Finish and go to Dashboard" : "Archive";
```

Comment:
The previous logic reads to me like this: In case updates are not allowed, I am the owner and others may not edit, I get the text "Copy To My Account" although I am the only person allowed to edit my own annotation?! Wouldn't "Duplicate" make much more sense here?

Therefore, I simplified the logic to whether the annotation is owned by the current user (-> "Duplicate") or not (-> "Copy to my account")

What do you think about my decision? Maybe I am missing something here :thinking:

Reply:
yes, this is good! this has annoyed me for some time now :)